src/app/capture_thread.cpp

```diff
@@ -38,10 +38,12 @@ CaptureThread::CaptureThread(int cam_id)
   captureModule->addItem("DC 1394");
   captureModule->addItem("Video 4 Linux");
   captureModule->addItem("Read from files");
+  captureModule->addItem("Flycapture");
```

Comment:
This should be done only if the FlyCap library has been found. For an example, please see [src/app/capture_thread:62](https://github.com/RoboCup-SSL/ssl-vision/blob/ecd668a8d5f1afc416f05aad34be4cb331e9244f/src/app/capture_thread.cpp#L62) to see how the optional mvImpact capture device is handled.